api/api_test.go

```diff
@@ -669,3 +772,30 @@ func assertClientSentVerb(t *testing.T, expectedVerb string, gotVerb string) {
 		t.Errorf("Expected request verb: %s, got %s", expectedVerb, gotVerb)
 	}
 }
+
+func encryptToArmor(t *testing.T, decryptedMessage string, pgpKey *pgpkey.PgpKey) (string, error) {
```

Comment:
here's the tech-debt card tracking that we need to combine this stuff across the client and API :) https://trello.com/c/UYrnfxJH